### ISSUE TITLE
Error?

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#1975]
_From @ZFM2004 on May 19, 2017 19:55_

When I Have A Turret Go On A Conveyor, it instantly dies. can you fix that?

_Copied from original issue: BEEmod/BEE2.4#635_

**Comment by Luke18033** [comment]
What do you mean by "dies", does it fizzle or just tip over?

**Comment by TeamSpen210** [comment]
Something on my todo list is to implement specific Turret Conveyor Belts - they would have turrets clamped down on the belt to prevent them falling over. Unless you block it of course though.

**Comment by Luke18033** [comment]
You mean Conveyor Belt Turrets? Turret Conveyor Belts would be kind of weird...

**Comment by TeamSpen210** [comment]
Yes, like the ones seen at the end of the Funnels Coop Course.

**Comment by Endermage77** [comment]
A Turret Conveyor belt actually sounds interesting.
It could spawn turrets facing a specific direction from the start with the conveyor pieces and destroy them if they reach the end.
This would have all the functionality of the regular conveyor belt in addition to having turrets.
Start Reversed would  flip the direction of the turrets.

**Comment by Luke18033** [comment]
Since it would only be placeable on the floor, why not add a 360-degree handle?

**Comment by Endermage77** [comment]
Well, when I said Turret Conveyor Belt, I implied that there would be a brand new item that was a conveyor belt that also spawned turrets.

**Comment by Luke18033** [comment]
That's what I was referring to, the original conveyor belt would still be 4-direction only.

**Comment by Luke18033** [comment]
I'm going to try adding a 360-degree handle to the track platform and see what happens.

**Comment by Luke18033** [comment]
Unfortunately the handle doesn't work - although it appears to, all the platforms end up facing the same direction when you compile. Additionally the full 360 rotation doesn't work for some reason, you can only rotate it in increments of 45 degrees.

**Comment by TeamSpen210** [comment]
Yeah, I didn't think it'd work. The item has to do some special calculations to figure out the right direction to set the movement direction to.
